.github/workflows/ci-server.yml

```diff
@@ -1,19 +1,7 @@
 name: CI (server)
 
-on:
-  # Trigger analysis when pushing in master or pull requests, and when creating
-  # a pull request, only if `server/` files are modified
-  push:
-    branches:
-      - master
-    paths:
-      - 'server/**'
-      - '.github/workflows/ci-server.yml'
-  pull_request:
-    types: [opened, synchronize, reopened]
-    paths:
-      - 'server/**'
-      - '.github/workflows/ci-server.yml'
```

Comment:
I think you could let that as is: as you modified the ci-server.yml file, it should trigger the job